app/services/quotes_service.rb

```diff
@@ -0,0 +1,11 @@
+class QuotesService
+  def self.conn
+    Faraday.new(url: 'https://api.quotable.io/')
+  end
+
+  def self.searched_quotes(keyword)
+    response = conn.get('/search/quotes', { query: keyword, limit: 150 })
```

Comment:
what are you limiting here? I would probably limit this to what the story specifies which is 10. Nice job limiting here at the call.